media-api/test/lib/elasticsearch/ElasticSearchTestBase.scala

```diff
@@ -20,8 +20,21 @@ class ElasticSearchTestBase extends FunSpec with BeforeAndAfterAll with Matchers
     createImage(UUID.randomUUID().toString, Handout(), usages = List(createDigitalUsage())),
 
     // with user metadata
-    createImage(id = "test-image-13-edited", Handout()).copy(userMetadata = Some(Edits(metadata = ImageMetadata(credit = Some("author")))), uploadTime = DateTime.now.minusMonths(1)),
-    createImage(id = "test-image-14-unedited", Handout()).copy(uploadTime = DateTime.now.minusMonths(1)),
+    createImage(id = "persisted-because-edited", Handout()).copy(
+      userMetadata = Some(Edits(
+        metadata = ImageMetadata(credit = Some("author")))
+      ),
+      uploadTime = DateTime.now.minusMonths(1)
```

Comment:
would it be worth extracting the upload time to a constant that clarifies that it's an irrelevant date in the past / a specific date in the past?

Reply:
👍 went a bit further and created a helper function [`createImageUploadedInThePast`](https://github.com/guardian/grid/pull/2508/files#diff-ad7c3b07522d48cba07d084c72fd1ec3R58).